addons/edi/security/res_groups.xml

```diff
@@ -81,5 +81,30 @@
             <field name="implied_ids" eval="[(4, ref('edi.group_edi_gateway_create'))]"/>
         </record>
 
+        <!-- EDI Transfers -->
+        <record id="group_edi_transfer_view" model="res.groups">
+            <field name="name">View Only</field>
+            <field name="category_id" ref="edi.module_category_edi_transfer"/>
+            <field name="implied_ids" eval="[(4, ref('edi.group_edi_gateway_view'))]" />
+        </record>
+
+        <record id="group_edi_transfer_edit" model="res.groups">
+            <field name="name">View &amp; Edit</field>
+            <field name="category_id" ref="edi.module_category_edi_transfer"/>
+            <field name="implied_ids" eval="[(4, ref('edi.group_edi_transfer_view'))]" />
+        </record>
+
+        <record id="group_edi_transfer_create" model="res.groups">
```

Comment:
Can we add this group to the 4 buttons on the view? So the user will only see them if they can press without getting an error.